app/controllers/practice_resources_controller.rb

```diff
@@ -0,0 +1,41 @@
+class PracticeResourcesController < ApplicationController
+  before_action :authenticate_user!, except: [:download]
+  before_action :set_practice_resource, only: [:download]
+
+  def download
+    if @practice_resource
+      redirect_to @practice_resource.attachment_s3_presigned_url
+    else
+      render plain: "Resource not found", status: :not_found
```

Comment:
Not sure if we want some kind of message here notifying the user to email the practice admin?